app/models/badges/researcher_badge.rb

```diff
@@ -0,0 +1,40 @@
+module Badges
+  class ResearcherBadge < Badge
+    seed "Researcher",
```

Comment:
I was torn between naming this `Researcher` and naming this `Scientist`. Let me know which you prefer.

Reply:
How about perquisitor? or seeker?  Or tryer-outer or maybe zetitic.